packages/nimble-components/src/anchor-tabs/tests/anchor-tabs-matrix.stories.ts

```diff
@@ -37,6 +37,7 @@ const component = (
         ${when(() => toolbar, html`
             <${tabsToolbarTag}>
                 <${buttonTag} appearance="ghost">Toolbar Button</${buttonTag}>
+                <${buttonTag} appearance="ghost" style="margin-left: auto;">Right-aligned Button</${buttonTag}>
```

Comment:
hmmm should our tabs toolbar align better with the toolbar and support the end slot idea natively?

Was wondering if we should get rid of tabs-toolbar altogether and have an appearance mode on toolbar but I guess they behave pretty different and it's not clear that the normal toolbar interactions are preferred:
Toolbar:
- start, default, and end slots all available
- uses arrow keys to move between items
- has own aria role defined

Tabs toolbar:
- only default slot
- just does simple layout of elements, no custom keyboard interactions
- no aria role defined (maybe fine since it's kinda transparent, the buttons slotted in in are just next in dom order anyway)

Scanning other usages in systemlink:
- Would have been nice to specify standard gap between items slotted in toolbar
- See use cases for specifying a start and end but maybe a default / middle is useful? I can't tell what's going on in [this UI](https://dev.azure.com/ni/DevCentral/_git/Skyline?path=/Web/Workspaces/LabManagement/src/app/work-orders/components/work-order-details/work-order-details.component.scss&version=GBmaster&line=38&lineEnd=39&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) or where to find it deployed.

If it seems like it's useful for the tabs-toolbar to have more slots styled, I'm assuming we wouldn't want to do it in this PR and could make a separate tech debt? (unless it does make sense to tackle now).

Reply:
You can see the UI you linked at `/labmanagement/workorders?ff-workorders=&ff-testplans=` and then click on any of the work orders in the grid to get to its details page. I didn't look closely at the CSS you linked but here's what the toolbar looks like.

![image](https://github.com/ni/nimble/assets/10500124/d94312bc-7876-4bfa-b93b-c630dd6ae908)

Reply:
If we decide to change this, I agree it'd likely make sense as a follow up.

Reply:
@rajsite brought up valid questions, and it isn't clear to me either whether or not the usages of the `tabs-toolbar` today should behave like a toolbar from an accessibility point of view. I'd really like to get this fix in sooner rather than later since our apps have incorrect styling without it. I've created an issue to figure out the answer to these questions and implement it: #1385